api/src/cli/utils/drivers.ts

```diff
@@ -1,4 +1,6 @@
-export const drivers = {
+import { Driver } from '../../types';
+
+export const drivers: Record<Driver, string> = {
```

Comment:
Sorry, a bit unrelated but WDYT about usuing a `Map` here instead?

```ts
const driversMap: Map<string, Driver> = new Map([
	['PostgreSQL / Redshift', 'pg'],
	['CockroachDB (Beta)', 'cockroachdb'],
	['MySQL / MariaDB / Aurora', 'mysql'],
	['SQLite', 'sqlite3'],
	['Microsoft SQL Server', 'mssql'],
	['Oracle Database', 'oracledb'],
]);

// For "api/src/cli/commands/init/index.ts"
export const drivers = Array.from(drivers.keys());
```

This way the `getDriverForClient` function could be as simple as:
```ts
return driversMap.get(client) || null
```

---

This import here

https://github.com/directus/directus/blob/5ac28ab28b085063f2dc96eccfa5d979867691f2/api/src/cli/utils/create-env/index.ts#L7 should probably changed to the following anyway:
```ts
import { Driver } from '../../../types';
```

Reply:
Theoretically yes, but those `drivers` are also imported and used in other files, so I don't want to force that refactor through as part of this PR 👍🏻